### PR TITLE
TF-511 Add GetIdentityMethod and unit test for it

### PR DESCRIPTION
**Pull request by dab246**
@hoangdat  @johnnguyenit39 Please review it

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

File: lib/http/converter/identities/identity_id_nullable_converter.dart [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import 'package:jmap_dart_client/jmap/core/id.dart';
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
+import 'package:json_annotation/json_annotation.dart';
+
```
Comment (johnnguyenit39) [inline lib/http/converter/identities/identity_id_nullable_converter.dart:4]: Read it

File: lib/http/converter/identities/signature_nullable_converter.dart [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import 'package:jmap_dart_client/jmap/identities/identity.dart';
```
Comment (johnnguyenit39) [inline lib/http/converter/identities/signature_nullable_converter.dart:1]: Read it

File: lib/jmap/identities/get/get_identity_method.dart [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
```
Comment (johnnguyenit39) [inline lib/jmap/identities/get/get_identity_method.dart:1]: Read it

File: lib/jmap/identities/get/get_identity_response.dart [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import 'package:jmap_dart_client/http/converter/account_id_converter.dart';
```
Comment (johnnguyenit39) [inline lib/jmap/identities/get/get_identity_response.dart:1]: Read it

File: lib/jmap/identities/identity.dart [hunk 5]
```diff
@@ -0,0 +1,93 @@
+import 'package:equatable/equatable.dart';
```
Comment (johnnguyenit39) [inline lib/jmap/identities/identity.dart:1]: Read it

File: test/jmap/identities/get_identity_method_test.dart [hunk 6]
```diff
@@ -0,0 +1,118 @@
+import 'package:dio/dio.dart';
```
Comment (johnnguyenit39) [inline test/jmap/identities/get_identity_method_test.dart:1]: Read it

**Pull request merged**